Update OSCAR information in EC3 docs (#369)
EGI-Federation/documentation: content/en/users/cloud-compute/ec3/apps/_index.md

To deploy a virtual cluster with
[Marathon](https://mesosphere.github.io/marathon/),
[Mesos](http://mesos.apache.org/), and
[Chronos](https://mesos.github.io/chronos/) as an orchestration, please select
Mesos + Marathon + Chronos from the list of available LRMS.

### OSCAR cluster

To deploy
[Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
in EGI, please select OSCAR from the list of LRMS (Local Resource Management
System). OSCAR supports data-driven serverless computing for file-processing
applications. A file upload, to an object storage backend, will trigger the
execution of a chosen shell script running inside a user-defined container.

> These will be orchestrated as a Kubernetes batch jobs. The output data will be

Reviewer: ```suggestion
These will be orchestrated as Kubernetes batch jobs. The output data will be
```

> uploaded to any object storage backends support. Synchronous invocations

Reviewer: ```suggestion
uploaded to any object storage that the backends support. Synchronous invocations are
```

Still it's a bit fuzzy how the backend relates to the object storage, probably this is detailed in the OSCAR docs, but here it's confusing. Maybe this whole sentence should be simplified like (just a suggestion, please improve as needed) and the details can be then read at OSCAR's docs

```
File uploads to a supported object storage will trigger the execution of actions inside user-defined
containers that are orchestrated as Kubernetes batch jobs. Output of those actions can be uploaded 
to object storage as well.
```

Author: I have updated the description. Please check if it is clearer now.

To deploy a virtual cluster with
[Marathon](https://mesosphere.github.io/marathon/),
[Mesos](http://mesos.apache.org/), and
[Chronos](https://mesos.github.io/chronos/) as an orchestration, please select
Mesos + Marathon + Chronos from the list of available LRMS.

### OSCAR cluster

To deploy
[Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
in EGI, please select OSCAR from the list of LRMS (Local Resource Management
System). OSCAR supports data-driven serverless computing for file-processing
applications. A file upload, to the object storage backend [MinIO](http://minio.io),

> will trigger the execution of a chosen shell script running inside a user-defined

Reviewer: maybe just a nitpick, only shell scripts can be run?

Author: Yes, it must be a shell script (than can execute anything that is inside the container image)

container. These will be orchestrated as Kubernetes batch jobs. The output data
will be uploaded to any object storage backends support. Synchronous invocations
are also available.

As external object storage providers, the following services can be used:

> - External [MinIO](https://min.io) servers, which may be in clusters other than the 

Reviewer: ```suggestion
- External [MinIO](https://min.io) servers, which may be in clusters other than the
```

  platform.
- [Amazon S3](https://aws.amazon.com/s3/), the Amazon's  object storage service that
  offers industry-leading scalability, data availability, security, and performance
  in the public Cloud.
- [Onedata](https://onedata.org/), the global data access solution for science used

>   in the [EGI Federated Cloud](https://datahub.egi.eu/).

Reviewer: ```suggestion
  in the [EGI DataHub](https://datahub.egi.eu/).
```

To deploy
[Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
in EGI, please select OSCAR from the list of LRMS (Local Resource Management
System). OSCAR supports data-driven serverless computing for file-processing applications. 

> Services will be triggered in response to a file upload to an object storage backend in order 
> to execute a user-defined shell script inside a container provisioned out of an user-defined 
> Docker image. These will be orchestrated as a Kubernetes batch jobs. The output data will be uploaded 

Reviewer 2: This sentence seems a bit too convoluted/complex, maybe we could try to simplify in a way similar the the proposition below (I may have misunderstood something, so please adapt):
```suggestion
A file upload, to an object storage backend, will trigger the execution of a chosen shell script running inside a user-defined container. These will be orchestrated as a Kubernetes batch jobs. The output data will be uploaded 
```
